### PR TITLE
FIX: fix AttributeError Xtick has no attribute label

### PR DESCRIPTION
**Pull request by celprov**

### CODE REVIEW

File: nireports/reportlets/xca.py [hunk 1]
```diff
@@ -359,8 +359,8 @@ def compcor_variance_plot(
         ax[m].set_yticks([])
         ax[m].set_yticklabels([])
         for tick in ax[m].xaxis.get_major_ticks():
```
Comment (effigies) [inline nireports/reportlets/xca.py:361]: This feels cleaner to me:

```suggestion
        for label in ax[m].xaxis.get_majorticklabels():
```

File: nireports/reportlets/xca.py [hunk 2]
```diff
@@ -359,8 +359,8 @@ def compcor_variance_plot(
...
-            tick.label.set_fontsize("x-small")
-            tick.label.set_rotation("vertical")
+            tick.label1.set_fontsize("x-small")
+            tick.label1.set_rotation("vertical")
```
Comment (effigies) [inline nireports/reportlets/xca.py:363]:
```suggestion
            label.set_fontsize("x-small")
            label.set_rotation("vertical")
```

**Review by effigies — COMMENTED** [review]
Suggestion. This mirrors https://github.com/nipreps/niworkflows/pull/820. Assuming we also have `confounds_correlation_plot` and `plot_melodic_components`, we should update those as well.

**Comment by celprov** [comment]
@effigies The three functions that I changed (`compcor_variance_plot`, `plot_melodic_components` and `confounds_correlation_plot` ) should already covered by tests [here](https://github.com/nipreps/nireports/blob/aa620edf93a0003bfc59a5126eb018b0a9b94a62/nireports/tests/test_reportlets.py#L177), [here](https://github.com/nipreps/nireports/blob/aa620edf93a0003bfc59a5126eb018b0a9b94a62/nireports/tests/test_reportlets.py#L256C15-L256C15) and [here](https://github.com/nipreps/nireports/blob/aa620edf93a0003bfc59a5126eb018b0a9b94a62/nireports/tests/test_interfaces.py#L49C5-L49C5).

**Comment by effigies** [comment]
Oh good. I didn't realize that the missing coverage was covered by the tests Taylor just added.

**Comment by celprov** [comment]
@effigies, but why is codecov complaining that the new lines are not covered by the test then?

**Comment by effigies** [comment]
Hmm. I thought the branches just weren't merged together, but it looks like the merge commit also didn't see these as covered: https://app.codecov.io/gh/nipreps/nireports/commit/9f62aca1759406934157255ebb4a66edd1bc8ea0

**Pull request merged**